tests/core/gromov_wasserstein_test.py

```diff
@@ -231,6 +233,25 @@ def test_gw_lr_fused(self):
     self.assertGreater(
       jnp.linalg.norm(ot_gwlr.matrix - ot_gwlreps.matrix), 1e-3)
 
+  @parameterized.parameters([True, "mean", "max_cost"])
+  def test_gw_fused_scale_cost(self, scale_cost):
+    epsilon = 0.1
+    fused_penalty = 1
+    geom_x = pointcloud.PointCloud(self.x, scale_cost=None)
+    geom_y = pointcloud.PointCloud(self.y, scale_cost=None)
+    geom_xy = pointcloud.PointCloud(self.xx, self.yy, scale_cost=None)
+    geom_x_scaled = pointcloud.PointCloud(self.x, scale_cost=scale_cost)
+    geom_y_scaled = pointcloud.PointCloud(self.y, scale_cost=scale_cost)
+    geom_xy_scaled = pointcloud.PointCloud(self.xx, self.yy, scale_cost=scale_cost)
+
+    gt = gromov_wasserstein.gromov_wasserstein(geom_xx=geom_x_scaled, geom_yy=geom_y_scaled, geom_xy=geom_xy_scaled,
```

Comment:
80 chars (here and elsewhere)

Reply:
Done, although in #43 I will argue to go for `120`.